contrib/google.golang.org/grpc/appsec_test.go

```diff
@@ -378,7 +379,7 @@ func (s *appsecFixtureServer) StreamPing(stream Fixture_StreamPingServer) (err e
 	md, _ := metadata.FromIncomingContext(ctx)
 	ids := md.Get("user-id")
 	if err := pappsec.SetUser(ctx, ids[0]); err != nil {
-		if e, ok := err.(pappsec.MonitoringError); ok {
+		if e, ok := err.(*grpcsec.MonitoringError); ok {
 			err = status.Error(codes.Code(e.GRPCStatus()), e.Error())
 		}
```

Comment:
Blocking should be automatically done by our middleware and not manually by the handler itself:
```suggestion
```

Reply:
This is the blocking in SDK case, similar to how we ask users to interrupt their handlers in case of an error for http.

Reply:
Fixed in [905cd39](https://github.com/DataDog/dd-trace-go/pull/2044/commits/905cd396e9fdc970f2e125216c2ce8479450939a)